javascripts/background_scripts/test_loader.js

```diff
@@ -26,7 +26,7 @@ function loadJs(filename){
 function getMetaValue(metaName){
   var metas = document.getElementsByTagName("meta");
   for (var i = 0; i < metas.length; i++){
-    if(metas[i].getAttribute('name') == metaName){
+    if(metas[i].getAttribute('name') === metaName){
```

Comment:
The testing parts should be on a different commit or not included at all. The way we are running unit tests outside privly-applications is also outdated -- as in they are not integrated with Karma. This is something we need to open an issue for...